Code/GraphMol/ChemReactions/ReactionFingerprints.h

```diff
@@ -40,9 +40,9 @@ namespace RDKit {
 
 class ChemicalReaction;
 
-enum FingerprintType {
+enum class FingerprintType {
   AtomPairFP = 1,
-  TopologicalTorsion,
+  TopologicalTorsionFP,
```

Comment:
Not sure if it is worth doing this; I am putting in this comment just in case you forgot. If you are happy to change the value directly given it only affects C++ users, I am happy too.
```suggestion
  TopologicalTorsionFP,
  // deprecated
  TopologicalTorsion = TopologicalTorsionFP,
```

Reply:
I figured that this is an obscure enough bit of code that this would be ok